app/services/fast_track/hypertension_medication_request_data.rb

```diff
@@ -32,8 +32,8 @@ def get_notes_from_note(verbose_notes)
     end
 
     def get_text_from_dosage_instruction(dosage_instructions)
-      toplevel_texts = dosage_instructions.map { |instr| instr['text'] }
-      code_texts = dosage_instructions.map { |instr| instr['timing']['code']['text'] }
+      toplevel_texts = dosage_instructions.map { |instr| instr['text'] || [] }
```

Comment:
Should we dig here too?

Reply:
No need, if `'text'` doesn't exist here it will return `[]`.